writers.go

```diff
@@ -362,6 +369,32 @@ func (w *UserTypesWriter) Execute(data *UserTypeTemplateData) error {
 	return w.ExecuteTemplate("types", userTypeT, fm, data)
 }
 
+// NewStorageGroupsWriter returns a contexts code writer.
+// Storage group types contain custom access groups as defined in the DSL with "StorageGroup".
+func NewStorageGroupsWriter(filename string) (*StorageGroupsWriter, error) {
+	file, err := codegen.SourceFileFor(filename)
+	if err != nil {
+		return nil, err
+	}
+	return &StorageGroupsWriter{SourceFile: file}, nil
+}
+
+// Execute writes the code for the context types to the writer.
+func (w *StorageGroupsWriter) Execute(data *StorageGroupDefinition) error {
+	fm := make(map[string]interface{})
```

Comment:
Fixed in https://github.com/goadesign/gorma/pull/105/commits/6351f7205e964adab3e8175822fafc058fd3cf1b